File: system-test/dotnet/SystemTests/Latest/SmokeTests/External/ErpSmokeTest.cs
using SystemTests.Latest.Base;
using Xunit;

namespace SystemTests.Latest.SmokeTests.External;

public class ErpSmokeTest : BaseScenarioDslTest
{
    [Fact]
    public async Task ShouldBeAbleToGoToErp()
    {
        await Scenario().Assume().Erp().ShouldBeRunning();
    }
}

File: system-test/java/src/test/java/com/optivem/eshop/systemtest/latest/smoke/external/ErpSmokeTest.java
package com.optivem.eshop.systemtest.latest.smoke.external;

import com.optivem.eshop.systemtest.latest.base.BaseScenarioDslTest;
import org.junit.jupiter.api.Test;

class ErpSmokeTest extends BaseScenarioDslTest {
    @Test
    void shouldBeAbleToGoToErp() {
        scenario.assume().erp().shouldBeRunning();
    }
}


